tests/em/fdem/muinverse/test_muinverse.py

```diff
@@ -177,19 +178,21 @@ def fun(x):
                 lambda x: self.simulation.Jvec(mod, x),
             )
 
-        dx = np.random.rand(*mod.shape) * (mod.max() - mod.min()) * 0.01
+        rng = np.random.default_rng(seed=3321)
+        dx = rng.uniform(size=mod.shape) * (mod.max() - mod.min()) * 0.01
 
-        return tests.check_derivative(fun, mod, dx=dx, num=3, plotIt=False)
+        np.random.seed(1983)  # set a random seed for check_derivative
+        return tests.check_derivative(fun, mod, dx=dx, num=4, plotIt=False)
```

Comment:
I increased the number of iterations here. Basically, the perturbations were a bit too big for us to see the second order behaviour, but by adding 1 (these ones are fast), you can see that the derivatives are indeed second order.

Reply:
Great! Thanks @lheagy!